src/next-child-view-container.js

```diff
@@ -0,0 +1,108 @@
+import _ from 'underscore';
+import emulateCollection from './utils/emulate-collection';
+
+// Provide a container to store, retrieve and
+// shut down child views.
+const Container = function() {
+  this._views = [];
```

Comment:
Yeah.. I wish I had started there.. unfortunately I went the long way around starting with having the "buffer" in the collectionView available all of the time.. and then indexing it..and a bunch of other bad ideas before it was clear it was better to just do it in the container..  which enumerable knew all along.  @ahumphreys87 does anyone use this lib?  Maybe we should deprecate babysitter and point people to that lib.